runtime/caml/signals.h

```diff
@@ -44,13 +44,11 @@ void caml_record_signal(int signal_number);
 void caml_set_something_to_do (void);
 value caml_check_gc_without_async_callbacks(value root);
 void caml_raise_in_async_callback (value exc);
-void caml_process_event(void);
 int caml_set_signal_action(int signo, int action);
 
 CAMLextern void (*caml_enter_blocking_section_hook)(void);
 CAMLextern void (*caml_leave_blocking_section_hook)(void);
 CAMLextern int (*caml_try_leave_blocking_section_hook)(void);
-CAMLextern void (* volatile caml_async_action_hook)(void);
```

Comment:
Indeed I did grep my local copy of the opam repository, but I did not mention it because the result were inconclusive. They were inconclusive because I did not find anything, and such a search can only show examples of uses, it cannot let us conclude that something is not used :) The argument is indeed that it's internal.

Reply:
Ok, I guess that should be enough checking.